frontend/src/component/segments/SegmentProjectAlert.tsx

```diff
@@ -100,10 +95,73 @@ export const SegmentProjectAlert = ({
     return null;
 };
 
-const formatStrategyNameParens = (strategy: IFeatureStrategy): string => {
+const formatStrategyNameParens = (strategy: {
+    strategyName?: string;
+}): string => {
     if (!strategy.strategyName) {
         return '';
     }
 
     return `(${formatStrategyName(strategy.strategyName)})`;
 };
+
+export const formatChangeRequestPath = (
+    projectId: string,
+    changeRequestId: number,
+): string => {
+    return `/projects/${projectId}/change-requests/${changeRequestId}`;
+};
+
+const strategyListItem = (
+    strategy:
+        | IFeatureStrategy
+        | ChangeRequestUpdatedStrategy
+        | ChangeRequestNewStrategy,
+    index: number,
+) => {
+    const isChangeRequest = (
+        strategy: IFeatureStrategy | ChangeRequestStrategy,
+    ): strategy is ChangeRequestStrategy => 'changeRequest' in strategy;
+
+    if (isChangeRequest(strategy)) {
+        const { id, title } = strategy.changeRequest;
+
+        const text = title ? `#${id} (${title})` : `#${id}`;
+        return (
+            <li key={`#${strategy.changeRequest.id}@${index}`}>
+                <p>
+                    {strategy.featureName}{' '}
+                    {`${formatStrategyNameParens(
+                        strategy,
+                    )} — in change request `}
+
+                    <Link
+                        to={formatChangeRequestPath(strategy.projectId, id)}
+                        target='_blank'
+                        rel='noopener noreferrer'
+                        title={`Change request ${id}`}
+                    >
+                        {text}
+                    </Link>
+                </p>
+            </li>
+        );
+    } else {
+        return (
+            <li key={strategy.id}>
+                <Link
+                    to={formatEditStrategyPath(
+                        strategy.projectId!,
+                        strategy.featureName!,
+                        strategy.environment!,
+                        strategy.id,
+                    )}
+                    target='_blank'
+                    rel='noopener noreferrer'
+                >
+                    {strategy.featureName!} {formatStrategyNameParens(strategy)}
+                </Link>
+            </li>
+        );
+    }
+};
```

Comment:
This is nearly identical to what we did in the segment delete component, except it uses `Link` instead of `StyledLink`. I wonder if it's worth extracting it or not. It feels weird duplicating it, but at the same time, they **are** two different lists that might not necessarily change together.